doc/source/user_guide/categorical.rst

```diff
@@ -1178,3 +1180,9 @@ Use ``copy=True`` to prevent such a behaviour or simply don't reuse ``Categorica
     This also happens in some cases when you supply a NumPy array instead of a ``Categorical``:
     using an int array (e.g. ``np.array([1,2,3,4])``) will exhibit the same behavior, while using
     a string array (e.g. ``np.array(["a","b","c","a"])``) will not.
+
+Counting CategoricalDtype
```

Comment:
Could you undo the changes in this file?

Reply:
Addressed [here](https://github.com/pandas-dev/pandas/pull/59327/commits/d8381812e305c6c60f5fde26ded4b907fe96db04)

Reply:
Looks like there's still changes in this file

Reply:
Ah sorry! Addressed [here](https://github.com/pandas-dev/pandas/pull/59327/commits/a847c134641a6e5dfa1c95e2e30bd6e67885b8cf)